about.html

```diff
@@ -0,0 +1,190 @@
+        </a>
+            
+        
+        </h2>
+      </div>
+
+    </section>
+    
+    <section id="headline2" 
+    class="w-100 text-center pt-5 pb-5">
+        <h2 id="head2-h2">Join Peers at the Advanced Battery and H/EV Technology Community</h2>
+        <div id="head2-line"></div>
+
+        <p id="head2-p1" class="d-none d-lg-block m-0">Join more than 770+ suppliers to showcase products representing the entire supply chain. This is the only trade fair where you can connect with over 10,000+ attendees in 3 days.This is the only trade fair where you can connect with over 10,000+ attendees in 3 days.</p>
+        <p id="head2-p2" class="m-0 mb-5">Join more than 770+ suppliers to showcase products representing the entire supply chain. </p>
+
+        <img width="80%" src="./imgs/electric-battery-logo2.png" alt="logo">  
+
+    </section>
+
+    <hr class="d-md-none">
+
+    <section id="past-conference" 
+    class="w-100 text-center">
+      <h5><b>See the past Event Summits</b></h5>
+      <div id="past-line"></div>
+      <p id="past-p" >have a look at EU's most important event in batteries and EV tech industries.</p>
+
+      <div id="video-container" class="row row-cols-1 row-cols-md-2 justify-content-between">
+        <div class="col">
+          <iframe width="100%" height="100%" src="https://www.youtube.com/embed/WhBTyulrktc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen
+          ></iframe>
+        </div>
+
+        <div class="col">
+          <iframe width="100%" height="100%" src="https://www.youtube.com/embed/L4bwJAt9iC8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+        </div>
+      </div>
+
+    </section>
+
+    <section id="social" class="w-100 bg-dark d-md-none">
+      <h2 id="soc-h2">Partner</h2>
+      <div id="soc-line"></div>
+      
+      <div class="d-flex flex-row flex-wrap justify-content-evenly">
+        <a id="soc-a" href="">
+          <span><img width="90px" src="./imgs/socials/mozilla.svg" alt=""></span></a>
+        <a id="soc-a" href="">
+          <span><img width="90px" src="./imgs/socials/googlekorea.svg" alt=""></span></a>
+        <a id="soc-a" href="">
+          <span><img width="90px" src="./imgs/socials/naver.svg" alt=""></span></a>
+        <a id="soc-a" href="">
+          <span><img width="90px" src="./imgs/socials/daumkakaologo.svg" alt=""></span></a>
+        <a id="soc-a" href="">
+          <span><img width="90px" src="./imgs/socials/airbnb.svg" alt=""></span></a>
+      </div>
+
+    </section>
+
+    <section id="contact" class="w-100 text-center">
+      <a id="con-a" href="#">
+        <span> <img src="./imgs/BatteryTech.png" alt="contact">
+        </span>
+        </a> 
+    </section>
```

Comment:
- [ ] Please make sure that the footer section looks appropriate on the desktop view and that the images don't take the full width with a different background. You can use [`filter invert`](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/invert) to achieve so.

![image](https://github.com/khanh-devos/Capstone1-conference/assets/62290545/abe6fff2-24eb-4c6d-8958-125c8856ae69)